development/build/scripts.js

```diff
@@ -1023,6 +1023,8 @@ async function getEnvironmentVariables({ buildTarget, buildType, version }) {
       environment,
       testing,
     }),
+    PORTFOLIO_URL:
+      ENVIRONMENT.PORTFOLIO_URL || 'https://portfolio.consensys.net',
```

Comment:
This should be https://portfolio.metamask.io/ right?

Reply:
Yup, let me update this.